backend/Squirrel.ConsoleApp/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models.Models;
+
+namespace Squirrel.Core.WebAPI.Extensions;
+
+public static class ApplicationBuilderExtensions
+{
+    private static HubConnection? _hubConnection;
+    private static string? _clientId;
+
+    public static void InitializeFileSettings(this IApplicationBuilder app)
+    {
+        using var scope = app.ApplicationServices.GetService<IServiceScopeFactory>()?.CreateScope();
+        var fileService = scope?.ServiceProvider.GetRequiredService<IConnectionFileService>();
+        fileService?.CreateEmptyFile();
```

Comment:
Why we need empty file?

Reply:
It was requirment at task [#54](https://trello.com/c/rKgwOm5u/54-as-a-developer-i-want-to-set-up-mechanism-to-pass-connection-string-to-console-app) @VladTkach
![image](https://github.com/BinaryStudioAcademy/bsa-2023-squirrel/assets/107005256/9bfd08db-a328-46e0-8acc-1bb48a0911f8)